scripts/check-build.js

```diff
@@ -20,20 +20,33 @@ function recompileFromSource() {
 }
 
 try {
-  console.log('@sentry/profiling-node: Precompiled binary found, attempting to load...');
-  require(target);
-  console.log('@sentry/profiling-node: Precompiled binary found, skipping build from source.');
+  if(!existsSync(target)) {
```

Comment:
Lgtm, do you mind just inverting these two conditions for readability and drop the negation? 

```
if(exists){
  require
} else {
  recompile
}
```

Reply:
yup! just pushed the requested changes